web/packages/teleport/src/Discover/Server/EnrollEc2Instance/Ec2InstanceList.tsx

```diff
@@ -0,0 +1,272 @@
+          <CommandBox
+            header={
+              <>
+                <Text bold>Configure your AWS IAM permissions</Text>
+                <Text typography="subtitle1" mb={3}>
+                  We were unable to list your EC2 instances. Run the command
+                  below on your AWS CloudShell to configure your IAM
+                  permissions. Then press the refresh button above.
+                </Text>
+              </>
+            }
+            showValidTtl={false}
+          >
+            <TextSelectCopyMulti
+              lines={[{ text: `bash -c "$(curl '${scriptUrl}')"` }]}
+            />
+          </CommandBox>
+        </Box>
+      )}
+    </>
+  );
+};
+
+// labelMatcher allows user to client search by labels in the format
+//   1) `key: value` or
+//   2) `key:value` or
+//   3) `key` or `value`
+function labelMatcher(
+  targetValue: any,
+  searchValue: string,
+  propName: keyof CheckedEc2Instance & string
+) {
+  if (propName === 'labels') {
+    return targetValue.some((label: Label) => {
+      const convertedKey = label.name.toLocaleUpperCase();
+      const convertedVal = label.value.toLocaleUpperCase();
+      const formattedWords = [
+        `${convertedKey}:${convertedVal}`,
+        `${convertedKey}: ${convertedVal}`,
+      ];
+      return formattedWords.some(w => w.includes(searchValue));
+    });
+  }
+}
+
+function RadioCell({
+  item,
+  isChecked,
+  onChange,
+  disabled,
+}: {
+  item: CheckedEc2Instance;
+  isChecked: boolean;
+  onChange(selectedItem: CheckedEc2Instance): void;
+  disabled: boolean;
+}) {
+  return (
+    <Cell width="20px" disabled={disabled}>
+      <Flex alignItems="center" my={2} justifyContent="center">
+        <input
+          // eslint-disable-next-line react/no-unknown-property
+          css={`
+            margin: 0 ${props => props.theme.space[2]}px 0 0;
+            accent-color: ${props => props.theme.colors.brand.accent};
+            cursor: pointer;
+
+            &:disabled {
+              cursor: not-allowed;
+            }
+          `}
+          type="radio"
+          name={item.awsMetadata.instanceId}
+          checked={isChecked}
+          onChange={() => onChange(item)}
+          value={item.awsMetadata.instanceId}
+          disabled={disabled}
+        />
+      </Flex>
+    </Cell>
+  );
+}
+
+const Labels = ({ labels }: { labels: Label[] }) => {
+  const $labels = labels.map((label, index) => {
+    const labelText = `${label.name}: ${label.value}`;
+
+    return (
+      <Pill key={`${label.name}${label.value}${index}`} mr="1" kind="secondary">
+        {labelText}
+      </Pill>
+    );
+  });
+
+  return <Flex flexWrap="wrap">{$labels}</Flex>;
+};
+
+const Cell: React.FC<{ disabled: boolean; width?: string }> = ({
+  disabled,
+  width,
+  children,
+}) => {
+  return (
+    <TableCell
+      width={width}
+      title={
+        disabled
+          ? 'this EC2 instance is already enrolled and is a part of this cluster'
```

Comment:
Capitalized the help messsage?

```suggestion
          ? 'This EC2 instance is already enrolled and is a part of this cluster'
```